libcudacxx/include/cuda/std/__utility/integer_sequence.h

```diff
@@ -198,12 +196,12 @@ struct _LIBCUDACXX_TEMPLATE_VIS integer_sequence
 template <size_t... _Ip>
 using index_sequence = integer_sequence<size_t, _Ip...>;
 
-#  if __has_builtin(__make_integer_seq) && !defined(_LIBCUDACXX_TESTING_FALLBACK_MAKE_INTEGER_SEQUENCE)
+#if __has_builtin(__make_integer_seq) && !defined(_LIBCUDACXX_TESTING_FALLBACK_MAKE_INTEGER_SEQUENCE)
```

Comment:
We centralize all builtin detection in `__config` could you please add a `_LIBCUDACXX_MAKE_INTEGER_SEQUENCE` macro that is defined when the builtin is available and use that here?

The idea here is that some compilers (cough MSVC) need some manual treatment at times, e.g.

```cpp
#  if __check_builtin(builtin_bit_cast) || (defined(_CCCL_COMPILER_MSVC) && _MSC_VER > 1925)
#    define _LIBCUDACXX_BIT_CAST(...) __builtin_bit_cast(__VA_ARGS__)
#  endif // __check_builtin(builtin_bit_cast)
```